tests/test_fetch_patch_step.py

```
@@ -0,0 +1,154 @@
+
+    # Verify progress comment was emitted
+    mock_emit.assert_called_once()
+
+
+@patch("rouge.core.workflow.steps.fetch_patch.emit_progress_comment")
+@patch("rouge.core.workflow.steps.fetch_patch.fetch_pending_patch")
+@patch("rouge.core.workflow.steps.fetch_patch.fetch_issue")
+def test_fetch_patch_step_no_pending_patch(
+    mock_fetch_issue,
+    mock_fetch_patch,
+    mock_emit,
```

Ruff (0.14.11)

Unused function argument: `mock_emit`

(ARG001)